ArduSub/GCS_Mavlink.cpp

```diff
@@ -1631,3 +1613,9 @@ bool GCS_MAVLINK_Sub::set_mode(uint8_t mode)
 
 // dummy method to avoid linking AFS
 bool AP_AdvancedFailsafe::gcs_terminate(bool should_terminate) { return false; }
+
+
+const AP_FWVersion &GCS_MAVLINK_Sub::get_fwver() const
```

Comment:
This in the wrong commit (*Sub: move handling of do_send_banner up*), should be in *Sub: add constant structure holding firmware version* like the other vehicles.

Reply:
Fixed.